scenario_player/runner.py

```diff
@@ -327,10 +327,10 @@ def _initialize_nodes(
         self.node_controller.initialize_nodes()
         node_addresses = self.node_controller.addresses
         node_count = len(self.node_controller)
-        node_balances = {address: self.client.balance(address) for address in node_addresses}
+        balance_per_nodes = {address: self.client.balance(address) for address in node_addresses}
```

Comment:
Shouldn't it be `node` instead of `nodes`? At least that's what you used everywhere else and it's correct English.

Reply:
Yupp, you're right - missed the `s` when replacing (`node_balance` instead of `node_balances`) so that was never removed. fixed!